docker_templates/templates/docker_images/create_drcsim_image.Dockerfile.em

```diff
@@ -18,24 +18,46 @@
     os_name=os_name,
     os_code_name=os_code_name,
 ))@
+@{
+packages = [
+    'dirmngr',
+    'gnupg2',
+    'lsb-release'
+]
+if 'upstream_packages' in locals():
+    if isinstance(upstream_packages, list):
+        for pkg in upstream_packages:
+            if pkg not in packages:
+                packages.append(pkg)
+}@
+@
+@[if packages != []]@
+
+# install packages
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    @(' \\\n    '.join(packages))@ \
+    && rm -rf /var/lib/apt/lists/* \
+    && mkdir /var/lib/apt/lists/partial
 
+@[end if]@
+@
 # setup keys
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
 
 # setup sources.list
-RUN echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-latest.list
+RUN . /etc/os-release \
```

Comment:
Is sourcing `/etc/os-release` still necessary when using `lsb_release`?

Reply:
Yes as we use `$ID` below that is populated by `os-release`